MIT.md

```diff
@@ -1,10 +1,7 @@
-## Copyright 2021, [YOUR NAME]
+## Copyright 2021, [Osman Wako]
 
-###### Please delete this line and the next one
-###### APP TYPE can be a webpage/website, a web app, a software and so on
+Permission is hereby granted, free of charge, to any person obtaining a copy of this Hello-Microverse and associated documentation files, to deal in the Hello-Microverse without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Hello-Microverse, and to permit persons to whom the Hello-Microverse is furnished to do so, subject to the following conditions:
 
-Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Hello-Microverse.
```

Comment:
- [OPTIONAL] The link to your MIT license is broken. Kindly follow this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to properly set it up for your project. I marked it as optional because it was not mentioned by the previous reviewer. But it is highly recommended to set it up properly every project you build at Microverse.

![license](https://user-images.githubusercontent.com/26597432/219015412-b1d78b86-49fb-446a-a8af-55a50b8f56f1.png)